### PR TITLE
fix(address): fix the nrc721 not displayed in the assets list

### PR DESCRIPTION
**Pull request by Daryl-L**
https://github.com/Magickbase/ckb-explorer-public-issues/issues/568

### CODE REVIEW

File: src/pages/Address/AddressAssetComp.tsx [hunk 1]
```diff
@@ -159,7 +159,7 @@ export const AddressNRC721Comp = ({ account, isRGBPP }: { account: NRC721; isRGB
     <AddressAssetComp
       isRGBPP={isRGBPP ?? false}
       href={`/nft-collections/${collection?.typeHash}`}
-      property={!symbol ? '?' : `#${amount}`}
+      property={!symbol ? '?' : `#${sliceNftName(amount, 20)}`}
```
Comment (Keith-CY) [inline src/pages/Address/AddressAssetComp.tsx:162]: How about using `text-overflow: ellipsis` by css

Comment (Keith-CY) [reply src/pages/Address/AddressAssetComp.tsx:162]: Any update on this conversation?

Comment (Daryl-L) [reply src/pages/Address/AddressAssetComp.tsx:162]: > Any update on this conversation?

Forgot to push to GitHub, pushed just now.

Comment (Keith-CY) [reply src/pages/Address/AddressAssetComp.tsx:162]: Is `sliceNftName` still necessary after using the `text-overflow: ellipsis`?

Comment (Daryl-L) [reply src/pages/Address/AddressAssetComp.tsx]: > Is `sliceNftName` still necessary after using the `text-overflow: ellipsis`?

have changed in https://github.com/Magickbase/ckb-explorer-frontend/pull/289/commits/6e44be7d134a6c222acb2022bcc24b0fda8e0483

**Review by Keith-CY — APPROVED** [review]

**Comment by FrederLu** [comment]
https://ckb-explorer-frontend-in-magickbase-repo-h3q4ska0l-magickbase.vercel.app/address/ckt1qpvvtay34wndv9nckl8hah6fzzcltcqwcrx79apwp2a5lkd07fdxxqvxn7gje034tgxnku60dym868umlnapl3g3unawn
image

Now there is data under `Simple User Defined Token`, but you need to confirm that the displayed data is mostly `?`. Is it related to this modification?

**Comment by Daryl-L** [comment]
> https://ckb-explorer-frontend-in-magickbase-repo-h3q4ska0l-magickbase.vercel.app/nft-collections/0xcfd1fcdc7f3bf8486ca1bd84382ea6c16f6d6e0c8f03ba9b67e1c4d2296ff132 image
> 
> https://ckb-explorer-frontend-in-magickbase-repo-h3q4ska0l-magickbase.vercel.app/address/ckt1qpvvtay34wndv9nckl8hah6fzzcltcqwcrx79apwp2a5lkd07fdxxqvxn7gje034tgxnku60dym868umlnapl3g3unawn image
> 
> Now there is data under `Simple User Defined Token`, but we need to confirm that the displayed data is mostly `?`. Is it related to this modification? At the same time, there is no information about `asd` on the address.

No, I think not, because they are not displayed before. And the data from api is null for these informations
image

**Comment by FrederLu** [comment]
https://ckb-explorer-frontend-in-magickbase-repo-h3q4ska0l-magickbase.vercel.app/address/ckt1qpvvtay34wndv9nckl8hah6fzzcltcqwcrx79apwp2a5lkd07fdxxq0ecfzek8k40fkpgxfpm3d5qmcntnqvrpqjr90cd

Api: https://ckb-explorer-api-staging.magickbase.com/api/v1/addresses/ckt1qpvvtay34wndv9nckl8hah6fzzcltcqwcrx79apwp2a5lkd07fdxxq0ecfzek8k40fkpgxfpm3d5qmcntnqvrpqjr90cd
image

The data of `udt type: nrc 721 token` and `udt type: m_nft_token` in the Staging API are different, and now both display `?`.

**Comment by Keith-CY** [comment]
> https://ckb-explorer-frontend-in-magickbase-repo-h3q4ska0l-magickbase.vercel.app/address/ckt1qpvvtay34wndv9nckl8hah6fzzcltcqwcrx79apwp2a5lkd07fdxxq0ecfzek8k40fkpgxfpm3d5qmcntnqvrpqjr90cd
> 
> 
> 
> Api: https://ckb-explorer-api-staging.magickbase.com/api/v1/addresses/ckt1qpvvtay34wndv9nckl8hah6fzzcltcqwcrx79apwp2a5lkd07fdxxq0ecfzek8k40fkpgxfpm3d5qmcntnqvrpqjr90cd
> 
> image
> 
> 
> 
> The data of `udt type: nrc 721 token` and `udt type: m_nft_token` in the Staging API are different, and now both display `?`.
> 
> 
> 
> Backend colleagues need to confirm the interface data.  @zmcNotafraid  @rabbitz 
> 
> 

Any update on this conversation @Daryl-L

**Review by FrederLu — APPROVED** [review]

**Pull request merged**